lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -27,6 +27,78 @@ class WP_Font_Library {
 		'woff2' => 'font/woff2',
 	);
 
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	public static $collections = array();
```

Comment:
I think this should be `private`, not `public`. There are public methods for interacting with this static property.

Reply:
Done!